src/sass/_variables.scss

```diff
@@ -0,0 +1,54 @@
+
+$theme-colors: (
+
+    'grey'              : #eeeeee,
+
+    'darkgrey'          : #A9A9A9,
```

Comment:
Secondly, the color/variable names should be better. Eg: primary, secondary etc

Reply:
The color names have been changed so that their use reflects the meaning.